openforcefield/topology/molecule.py

```diff
@@ -429,6 +435,24 @@ def is_bonded_to(self, atom2):
                     return True
         return False
 
+    @property
+    def is_in_ring(self):
+        """
+        Return whether or not this atom is in a ring(s) (of any size)
+
+        """
+        if self._molecule is None:
+            raise NotAttachedToMoleculeError(
```

Comment:
There are a few similar places that this exception could go (currently `ValueError`s)